ios/CodePush/RCTConvert+CodePushInstallMode.m

```diff
@@ -1,5 +1,10 @@
 #import "CodePush.h"
+
+#if __has_include("RCTConvert.h")
```

Comment:
I think this was more of an aesthetic preference than anything else (put the case we expect to use first), so there wasn't necessarily a specific technical reason for it.

With the duplicate imports, I'm not sure without additional investigation - but I could hypothesize that both `"RCTConvert.h"` and `<React/RCTConvert.h>` exist in 0.40 and later, and you're meant to use the latter.